contracts/lbp/LBPManagerFactory.sol

```diff
@@ -16,13 +16,14 @@ pragma solidity 0.8.17;
 
 import "../utils/CloneFactory.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
-import "./LBPManager.sol";
+import "./LBPManagerV1.sol";
 
 /**
  * @title LBPManager Factory
  * @dev   Governance to create new LBPManager contracts.
  */
 contract LBPManagerFactory is CloneFactory, Ownable {
+    bytes6 public version = "2.1.0";
```

Comment:
I expected to see "1.0.0" here, since LBP is still v1?

I also expect LBP and Seed versioning to be independent.
MAYBE we can find a case to tie the versions together, but I doubt that.